lib/utils/utils.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+void showSnackBar(BuildContext context, String content) {
```

Comment:
You can create some class ScreenUtils that have static methods like this one.

Reply:
and later you can use it on this way, example ScreenUtils.showSnackBar(context, 'Error happened')